extensions/adobe/experience/adcloud/searchads/adgroup.schema.json

```diff
@@ -5,39 +5,39 @@
     "you may not use this file except in compliance with the License. You may obtain a copy",
     "of the License at https://creativecommons.org/licenses/by/4.0/"
   ],
-  "$id": "https://ns.adobe.com/experience/adcloud/searchadvertising/adgroup",
+  "$id": "https://ns.adobe.com/xdm/adcloud/searchads/adgroup",
   "$schema": "http://json-schema.org/draft-06/schema#",
   "title": "Search Advertising AdGroup",
   "type": "object",
-  "meta:extensible": true,
-  "meta:abstract": true,
+  "meta:extensible": false,
+  "meta:abstract": false,
   "meta:auditable": true,
   "meta:extends": ["https://ns.adobe.com/xdm/data/record"],
   "description": "Search Advertising Ad Group is a logical collection or grouping of ads.",
   "definitions": {
-    "searchadvertising-adgroup": {
+    "searchads-adgroup": {
       "properties": {
-        "xdm:adgroupId": {
+        "searchads:adgroupId": {
           "title": "Ad Group Identifier",
           "type": "string",
           "description": "Ad Group ID on the search advertising platform."
         },
-        "xdm:campaignId": {
+        "searchads:campaignId": {
           "title": "Campaign Identifier",
           "type": "string",
           "description": "Campaign ID for the Ad Group on the search advertising platform."
         },
-        "xdm:internalCampaignId": {
+        "searchads:internalCampaignId": {
```

Comment:
What is the purpose of exposing `internalCampaignId`. None of the performance report uses `internalCampaignId` and neither the https://ns.adobe.com/xdm/adcloud/searchads/campaign schema has the `internalCampaignId`.

Reply:
The schemas are not defined only for reporting use cases. They are meant to be consumed by other data sources or internal solutions team.  As far as campaign is concerned we store internalCampaignId in the id field of the record object.  We are also storing internal ad group id in the id field.